src/lib/expansion-panel/expansion-panel-adapter.ts

```diff
@@ -20,12 +22,15 @@ export class ExpansionPanelAdapter extends BaseAdapter<IExpansionPanelComponent>
   private _headerElement: HTMLElement;
   private _contentElement: HTMLElement;
   private _headerSlotElement: HTMLSlotElement;
+  private _buttonElement: HTMLButtonElement | undefined;
+  private _ripple: ForgeRipple | undefined;
 
   constructor(component: IExpansionPanelComponent) {
     super(component);
     this._headerElement = getShadowElement(this._component, EXPANSION_PANEL_CONSTANTS.selectors.HEADER);
     this._contentElement = getShadowElement(this._component, EXPANSION_PANEL_CONSTANTS.selectors.CONTENT);
     this._headerSlotElement = getShadowElement(this._component, EXPANSION_PANEL_CONSTANTS.selectors.HEADER_SLOT) as HTMLSlotElement;
+    this._ripple = new ForgeRipple(this._headerElement);
```

Comment:
Haven't we been moving away from proactively creating the ripple element for performance reasons and waiting for user interaction to do so?  Also applies to the usage in `setType` below.

Reply:
I'm not even sure the ripple should be included in the component at all. It needs some more thought for sure.